lib/modules/translation/screens/translations.dart

```diff
@@ -0,0 +1,170 @@
+import 'package:andax/models/story.dart';
+import 'package:andax/models/translation.dart';
+import 'package:andax/modules/translation/screens/translation_editor.dart';
+import 'package:andax/modules/translation/services/translations.dart';
+import 'package:andax/modules/translation/widgets/translation_creator.dart';
+import 'package:andax/shared/services/story_loader.dart';
+import 'package:andax/shared/widgets/loading_dialog.dart';
+import 'package:andax/shared/widgets/rounded_back_button.dart';
+import 'package:flutter/material.dart';
+
+class TranslationsScreen extends StatefulWidget {
+  const TranslationsScreen({
+    required this.info,
+    Key? key,
+  }) : super(key: key);
+
+  final StoryInfo info;
+
+  @override
+  State<TranslationsScreen> createState() => _TranslationsScreenState();
+}
+
+class _TranslationsScreenState extends State<TranslationsScreen> {
+  List<StoryInfo>? translations;
+  late StoryInfo base;
+  late StoryInfo target;
+
+  @override
+  void initState() {
+    super.initState();
+    getAllTranslations(widget.info.storyID).then(
+      (s) => setState(() {
+        translations = s;
+        base = s.first;
+        target = base;
+      }),
+    );
+  }
+
+  void previewTranslation(StoryInfo info) {
+    showDialog<void>(
+      context: context,
+      builder: (context) {
+        return AlertDialog(
+          title: Text(info.title),
+          content: info.description.isEmpty ? null : Text(info.description),
+          actions: [
+            TextButton.icon(
+              onPressed: () {
+                setState(() => base = info);
+                Navigator.pop(context);
+              },
+              icon: const Icon(Icons.source_rounded),
+              label: const Text('As base'),
```

Comment:
It's not clear to me what these "as base" and "as target" mean. Is it "**Set** as base language" for the story? I thought we are going to treat all translations equally, without one being the "base" and others being translations. Only through trial and error did I figure out that this for the translation editing (making it irrelevant for the "adding" button). The icon for "base" is also not that demonstrative of the meaning.